unreal/src/lib.rs

```diff
@@ -2,11 +2,15 @@
 #![warn(missing_docs)]
 
 use std::fmt;
-use std::io::{self, Cursor, Read};
+use std::io::prelude::*;
+use std::io::{self, BufReader, Cursor, Read};
 
+use anylog::LogEntry;
 use bytes::{Buf, Bytes};
+use chrono::prelude::*;
```

Comment:
_nit_: We generally try to avoid star imports. Would it be possible to expand `std::io::prelude::*` and `chrono::prelude::*`?